scripts/update.js

```diff
@@ -215,43 +209,45 @@ const apply = (change) => {
   stats.changes += 1
 
   const name = change.id
-  const curr = packages[name]
+  const curr = packages.get(name)
 
   if (change.deleted) {
-    if (typeof curr === 'string') {
+    if (typeof curr !== 'undefined') {
```

Comment:
You can take advantage of the map: `package.has(name)`

Reply:
It looked strange to me to check with `packages.has()` and use the `curr` variable then. And replacing `curr` with `packages.get()` everywhere looked too wordy.